### ISSUE TITLE
/places issues

### ISSUE THREAD

**Issue by lostamongtrees** [RolePlayGateway/roleplaygateway#111]
![image](https://user-images.githubusercontent.com/52435234/60752408-3fbef980-9f93-11e9-8ffa-402bf2fd9c36.png)

fig1: if you have too many characters they appear in a block and push everything down the page, unsightly
fig2: not sure what's going on here, recently used?

Here is an example from a different roleplay, as I only had one character there it is more streamlined, but the row of locations still deems no purpose.
![image](https://user-images.githubusercontent.com/52435234/60752425-76950f80-9f93-11e9-9e32-938e840d7d78.png)

**Comment by lostamongtrees** [comment]
Should note it is also like that on the Universe front page
![image](https://user-images.githubusercontent.com/52435234/60764229-e074ee00-a053-11e9-8272-61d33672a06c.png)

**Comment by lostamongtrees** [comment]
Alright this is a little strange. So the places search works odd and i cant...seem to find the pattern to its  function at yet

![image](https://user-images.githubusercontent.com/52435234/61101284-38866880-a437-11e9-9d37-1f0baef66eda.png)


Other searches and their results

![image](https://user-images.githubusercontent.com/52435234/61101301-4936de80-a437-11e9-9584-b9f3889dadf4.png)

![image](https://user-images.githubusercontent.com/52435234/61101331-666bad00-a437-11e9-9bd5-08644c28ab66.png)

![image](https://user-images.githubusercontent.com/52435234/61101370-813e2180-a437-11e9-828e-8c2b80c6b413.png)

![image](https://user-images.githubusercontent.com/52435234/61101396-9450f180-a437-11e9-8af5-bec1051ba77e.png)

**Comment by NeverEndingFlip** [comment]
I can provide a little more insight as to what the issue is. Basically, it will only search from the list of places that are on the front page instead of from the entire list of places in the Multiverse. As shown in the screenshots above, it has no problem searching for places like Wing City, Gambit's Bar, or The Arena, but other places like Gaia, Ruula, and Blackrock cannot be found this way.

There is another issue that appears to be on the Multiverse Places tab. In the bottom right, it shows the list of pages you can visit, numbered from 1 to 2124 (which seems like an unusually high number). However, when you click on the next page to view, it only adds one new place to the front page at a time, rather than populate the entire page with new places, as it's supposed to do (I assume). This makes searching for new places to Roleplay a huge hassle as you have to individually search through anywhere from 20 to 2000 page refreshes before finding the place you need.

As I do not have much if any knowledge on coding, I do not know how to fix the issue. However, I hope the details prove helpful in pinpointing what the issue could be. :)

**Comment by lostamongtrees** [comment]
![image](https://user-images.githubusercontent.com/52435234/61176950-d2652700-a597-11e9-9d6e-b29216a88e5e.png)

**Comment by lostamongtrees** [comment]
This seeems fixed, as far as the places per page error

**Comment by martindale** [comment]
> Should note it is also like that on the Universe front page
> ![image](https://user-images.githubusercontent.com/52435234/60764229-e074ee00-a053-11e9-8272-61d33672a06c.png)

This applies to me as well, so I'm pretty sure it's limited to either A) beta, or B) GMs.  Maybe C) GMs in the Multiverse, but will need someone else to confirm they _don't_ see this.

**Comment by lostamongtrees** [comment]
I think it's GMs in The Multiverse via Beta specifically because other RPs don't seem to have it, and other GMs have yet to report the issue
